resolwe_bio/processes/import_data/annotation.yml

```diff
@@ -107,14 +101,26 @@
       igvtools index "${NAME}"_sorted.gff3
       re-checkrc "Indexing of GFF3 file failed."
 
-      flatfile-to-json.pl --gff "${NAME}"_sorted.gff3 --out . --trackLabel "annotation"
+      flatfile-to-json.pl \
+        --gff "${NAME}"_sorted.gff3 \
+        --out . \
+        --trackLabel "annotation" \
+        --nameAttributes "name,id,alias,gene_id,gene_name"
       re-checkrc "Annotation track processing for JBrowse failed."
 
+      # Prepare Seq folder
+      prepare-refseqs.pl --gff-sizes "${NAME}"_sorted.gff3 --out .
+      re-checkrc "Refseq track processing for JBrowse failed."
+
+      # Index features
+      generate-names.pl --out .
```

Comment:
Running the corresponding test with `resolwebio/rnaseq:4.9.0` Docker image I'm getting `No reference sequences defined in configuration, nothing to do.` The process fails in the next line.

Reply:
@mzganec `gt gff3validator` was replaced with `gt gff3 -tidy` mode, since the latter command reports and attempts to fix the file format errors, while the former command just reports the GFF3 format issues and does not stop the process, leading into issues downstream.